halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/ShowCommand.java

```diff
@@ -18,8 +18,11 @@
 
 import com.beust.jcommander.Parameter;
 import com.beust.jcommander.Parameters;
+import com.netflix.spinnaker.halyard.cli.execute.v1.ExecuteShow;
```

Comment:
My goal was to separate the structure of the commands (i.e. the `update` and `show` commands follows `config`) from how they are executed, because to me those are logically separate. You're right I didn't do a good job of keeping these structurally simple, let me revisit this Monday and show you a new implementation.